miranda/convert/data/ecmwf_cf_attrs.json

```diff
@@ -158,6 +314,22 @@
       "long_name": "10 metre northward wind",
       "standard_name": "northward_wind",
       "units": "m s-1"
+    },
+    "z": {
```

Comment:
Geopotential seems to only be available as an invariant surface param for era5-land..

Reply:
I think this means it would not be a `era5-single-levels`  variable ...  but instead a 4d variable. Something not implemented in miranda to date I think

Reply:
Geopotential isn't available for `era5-land` so no harm there. For `single-levels` there isn't any reason to grab anything more than one time-step as this doesn't change in time. I've added support for it since it was free.